### ISSUE TITLE
Doesn't work if Tab is pdf

### ISSUE THREAD

**Issue by aidan-gibson** [philc/vimium#4155]
moveTabToNewWindow doesn't work if current window is viewing a pdf
Vimium Version 1.67.2
Chrome Version 106.0.5249.119 (Official Build) (arm64)

**Comment by aidan-gibson** [comment]
Actually doesn't look like any of the extension works if a tab is a pdf. Haven't tested exhaustively.

**Comment by gdh1995** [comment]
Yes the bulitin PDF viewer plugin of Chrome (and the one of Firefox) refuses extensions to run code on it.

You may try the 3rd-party PDF viewer extension "PDF Viewer for Vimium C" (https://chrome.google.com/webstore/detail/pdf-viewer-for-vimium-c/nacjakoppgmdcpemlfnfegmlhipddanj) and it can work with my customized version of Vimium, Vimium C (https://github.com/gdh1995/vimium-c)

**Comment by reddylee** [comment]
> Yes the bulitin PDF viewer plugin of Chrome (and the one of Firefox) refuses extensions to run code on it.
> 
> You may try the 3rd-party PDF viewer extension "PDF Viewer for Vimium C" (https://chrome.google.com/webstore/detail/pdf-viewer-for-vimium-c/nacjakoppgmdcpemlfnfegmlhipddanj) and it can work with my customized version of Vimium, Vimium C (https://github.com/gdh1995/vimium-c)

PDF Viewer for Vimium C works for pdf but does not work in hypothesis.io. This one looks identical as hypothesis.io but without the annotation tools of hypothesis. Could you please improve this so it support hypothesis annotation? Thanks.

**Comment by philc** [comment]
Unfortunately, browsers don't let extensions run on their PDF viewer pages, so there's nothing Vimium can do. I've added an [FAQ entry](https://github.com/philc/vimium/wiki/FAQ#can-vimium-work-on-tabs-which-are-showing-pdf-files) about this.
